i18n/de-DE/account.json

```
@@ -0,0 +1,116 @@
+{
+  "account": {
+    "greeting": "Hallo {firstname} {lastname}",
+    "dropdownHeading": "Dein Konto",
+    "overview": {
+      "heading": "Übersicht",
+      "subHeading": "Hier siehst du deine Daten, die gespeicherte Bezahlmethode und Adressen.",
+      "personalInfo": {
+        "heading": "Persönliche Informationen",
+        "nameLabel": "Name:",
+        "emailLabel": "Email:"
+      },
+      "paymentMethod": {
+        "heading": "Bezahlmethode",
+        "nameLabel": "Name:",
+        "descriptionLabel": "Beschreibung:"
+      },
+      "billingAddressHeading": "Rechnungsadresse",
+      "shippingAddressHeading": "Lieferadresse",
+      "newsletter": {
+        "heading": "Newsletter Abonnement",
+        "inputLabel": "Ja, ich möchte den kostenlosen Newsletter erhalten. (Ich kann ihn jederzeit abbestellen)"
+      }
+    },
+    "profile": {
+      "heading": "Dein Profil"
+    },
+    "address": {
+      "heading": "Adressen"
+    },
+    "payment": {
+      "heading": "Bezahlmethoden"
+    },
+    "orders": {
+      "heading": "Bestellungen",
+      "noOrdersFoundInfo": "Whoops, es wurden keine Bestellungen gefunden. Hast du überhaupt schon was bestellt?",
+      "paymentStatusLabel": "Bezahlung",
+      "paymentMethodLabel": "Bezahlmethode",
+      "shippingStatusLabel": "Versand",
+      "shippingMethodLabel": "Versandmethode",
+      "lineItemsHeading": "Produkte",
+      "status": "Status"
+    },
+    "login": {
+      "successMessage": "Du wurdest erfolgreich eingeloggt.",
+      "errorMessage": "Es gab einen Fehler beim Login. Bitte versuch es noch einmal.",
+      "submitLabel": "Einloggen",
+      "emailInput": {
+        "label": "Email",
+        "placeholder": "quack{'@'}platsch.com",
+        "help": "Deine Emailadresse"
+      },
+      "passwordInput": {
+        "label": "Passwort"
+      },
+      "createAccountLink": "Erstelle hier deinen Account"
+    },
+    "register": {
+      "submitLabel": "Registrieren",
+      "dataHeading": "Deine Daten:",
+      "salutationInput": {
+        "label": "Anrede",
+        "placeholder": "Wähle deine Anrede",
+        "help": "Wähle aus, wie du angesprochen werden möchtest"
+      },
+      "firstnameInput": {
+        "label": "Vorname",
+        "placeholder": "Donald"
+      },
```

```suggestion
      "salutation": {
        "label": "Anrede",
        "placeholder": "Wähle deine Anrede",
        "help": "Wähle aus, wie du angesprochen werden möchtest"
      },
      "firstname": {
        "label": "Vorname",
        "placeholder": "Donald"
      },
```

could we not remove the input description here? i think its better to not reference an input or something like that (same issue for following snippets - also in other files)

---

I was thinking it would be good to be able to easily see what type of element the snippet refers to, that's why I chose to do it like this. Although that might be clear enough with the label and placeholders. Removing the suffix would probably make the snippet reusable in another context, is that why you think it would be better to remove it?

---

yeah exactly. i am hoping that if we choose the names more general, we will be able to reuse them